### ISSUE TITLE
qcost: use PBS options from a script

### ISSUE THREAD

**Issue by ccarouge** [coecms/nci_scripts#30]
It would be nice to have the possibility for qcost to read in the PBS directives from a script directly instead of having to give each value on the command line.
`qcost myscript.sh`

**Comment by aidanheerdegen** [comment]
You're right. That would be a nice feature, but would require parsing a the script, and there are different ways the options can be expressed, as well as having to take into account default values.
